blueprints/component-addon/index.js

```diff
@@ -11,13 +11,13 @@ module.exports = {
   fileMapTokens: function() {
     return {
       __path__: function(options) {
-        if (options.pod) {
+        if (options.pod && options.inRepoAddon) {
```

Comment:
Scratch that, this is the blueprint for component addons, not sure what's going on with this test currently.